src/svc/invocable-scripts/paths/scripts_scriptID.yml

```
@@ -4,33 +4,73 @@ get:
     - Data I/O endpoints
     - Invokable Scripts
   summary: Retrieve a script
-  description: Uses script ID to retrieve details of an invokable script.
+  description: |
+    Retrieve an invokable script.
```

```suggestion
    Retrieves an invokable script.
```

Endpoint should be the subject.